app/renderer/lib/client-frameworks/java.js

```diff
@@ -200,6 +201,8 @@ ${this.indent(code, 4)}
     return `Map<String, String> appStrings = driver.getAppStringMap(${language ? `${language}, ` : ''}${stringFile ? `"${stringFile}` : ''});`;
```

Comment:
since Java11, that we only support now we could same some space and use `var appStrings = ...`  syntax

Reply:
Alright, will change these occurrences elsewhere in the code as well